### DISCUSSION TITLE
RealTime : 1 user , 2 realtime tables = 2 concurrent connections?

### DISCUSSION THREAD

**Discussion by davidicon** [supabase/supabase#28789] [General]
Hi Good Morning - I am confuse with concurrent connections as subjected, 
I've two realtime tables "table_one" and "table_two"  
seems like need to subscribe both separately 
I seen 1 web socket is being used in browser console network tab, 
I want to know it will be considered 1 concurrent connection (based on websocket) or 2 ( subscribed table)

**Comment by GaryAustin1** [comment]
Connections are the websocket.  If you use supabase-js (or other supabase clients) then there is one websocket shared for many channels.
